apps/portal/prisma/schema.prisma

```diff
@@ -37,18 +37,27 @@ model Session {
 }
 
 model User {
-    id                  String               @id @default(cuid())
-    name                String?
-    email               String?              @unique
-    emailVerified       DateTime?
-    image               String?
-    accounts            Account[]
-    sessions            Session[]
-    todos               Todo[]
-    resumesResumes      ResumesResume[]
-    resumesStars        ResumesStar[]
-    resumesComments     ResumesComment[]
-    resumesCommentVotes ResumesCommentVote[]
+    id                           String                         @id @default(cuid())
+    name                         String?
+    email                        String?                        @unique
+    emailVerified                DateTime?
+    image                        String?
+    accounts                     Account[]
+    sessions                     Session[]
+    todos                        Todo[]
+    resumesResumes               ResumesResume[]
+    resumesStars                 ResumesStar[]
+    resumesComments              ResumesComment[]
+    resumesCommentVotes          ResumesCommentVote[]
+    QuestionsQuestion            QuestionsQuestion[]
+    QuestionsQuestionEncounter   QuestionsQuestionEncounter[]
+    QuestionsQuestionVote        QuestionsQuestionVote[]
```

Comment:
I think there's no need to add in references to votes and encounters from a user. References don't have to be bi-directional.

Reply:
Does this mean we don't want the foreign key constraint? Because I believe Prisma does this when we set a foreign key due to the specification of the Prisma client